setup.py

```diff
@@ -426,6 +426,17 @@ def make_deb():
     os.mkdir("./xpra-html5")
     shutil.copytree("./packaging/debian", "./xpra-html5/DEBIAN")
     install_html5("./xpra-html5/usr/share/xpra/www/", "uglifyjs")
+    # Move and symlink default-settings.txt
+    os.makedirs("./xpra-html5/etc/xpra/html5-client", exist_ok=True)
+    default_settings_files = { "default-settings.txt", "default-settings.txt.gz", "default-settings.txt.br" }
+    for filename in default_settings_files:
+        print("Checking existence of %s" % os.path.join("./xpra-html5/usr/share/xpra/www", filename))
+        if os.path.exists(os.path.join("./xpra-html5/usr/share/xpra/www", filename)):
+           os.rename(os.path.join("./xpra-html5/usr/share/xpra/www", filename),
+               os.path.join("./xpra-html5/etc/xpra/html5-client", filename))
+           os.symlink(os.path.join("/etc/xpra/html5-client", filename),
+               os.path.join("./xpra-html5/usr/share/xpra/www", filename))
```

Comment:
Why not just symlink this file from `www_dir` directly to `/etc/xpra/html5-client/`?
(and since it should be just one file - is the `html5-client` directory even needed?)

Reply:
The idea of whole change os that configuration files must reside in sub-directories of `/etc`, so symlink in `/usr/share/xpra/www` points to editable `/etc/xpra/default-settings.txt`. Separate directory for `html5-client` is not needed if we rename the file like `html5-client-settings.txt`.